feature-group-definitions/css-modules.dist.yml

```diff
@@ -0,0 +1,16 @@
+# Generated from: css-modules.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: CSS module scripts
+description: 'CSS module scripts allow CSS code to be organized into reusable units. Other modules use `import ... with {type: "css"}` to load CSS modules as constructable stylesheets.'
+spec: https://html.spec.whatwg.org/multipage/webappapis.html#css-module-script
+usage_stats: https://chromestatus.com/metrics/feature/timeline/popularity/3805
+status:
+  baseline: false
+  support:
+    chrome: "123"
```

Comment:
https://chromestatus.com/feature/5948572598009856 says Chrome 93, is that wrong?

Reply:
I believe that's when the old _assertion_ syntax (`assert`) got added. Since the new syntax relies on import _attributes_ instead (`with`), I think 123 is correct.

Reply:
See https://chromestatus.com/feature/5205869105250304

Reply:
That makes sense. It's akin to the feature being supported with another name or with a prefix, which we don't count as supported for web-features.